playground/.sassrc

```diff
@@ -0,0 +1,5 @@
+{
+	"includePaths": [
+		"node_modules"
```

Comment:
Needed for `~@wordpress/base-styles/colors` -style imports to work in the playground.

See https://github.com/parcel-bundler/parcel/issues/39

Reply:
Does a similar setup work for people consuming of the package via `webpack`/`sass-loader`?

How do people consume the package using something like `create-react-app`? `@import "node_modules/@wordpress/base-styles/colors`?

Reply:
`~` [is a Webpack thing](https://webpack.js.org/loaders/sass-loader/#resolving-import-at-rules) and Playground uses [Parcel](https://parceljs.org/scss.html). If `~` in consumer's build pipeline doesn't resolve to `node_modules`, they'd have to import using `node_modules/@wordpress/base-styles/colors` directly indeed. I'll mention that in docs.

Thanks for asking!

Reply:
The reason I ask is because the _default_ setup for `~` in `webpack`/`sass-loader` compiles any imports which means variables aren't accessible in the consuming file.

e.g. 
`node_modules/abc/index.scss`
```scss
$foo: 'bar';
```

`test.scss`
```scss
@import "~abc/index";

Reply:
> The reason I ask is because _from memory_ the _default_ setup for `~` in `webpack`/`sass-loader` is that `webpack` resolves and compiles any imports which means any variables/mixins defined in the imported file aren't accessible in the consuming file.

No, they do work. :-) I guess it could depend on your Webpack config tho?

I also updated README.md about `~`, wanna check if it's clear now?